pom.xml

```diff
@@ -2,7 +2,7 @@
 <!--
  DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
 
- Copyright 2011-2015 ForgeRock AS.
+ Copyright 2011-2016 ForgeRock AS.
```

Comment:
Is updating FR copyright entries something we should be doing? Shouldn't we just leave them alone?

Reply:
@siepkes ForgeRock actually updated it. This was constructed from their sources, to form 2.0.10.